### ISSUE TITLE
Feature request/idea?

### ISSUE THREAD

**Issue by LuOwl3** [alxscms/homebridge-somfy-protect#18]
Is it possible that the protect could be handle as some kind of bridge so that every device connected can get exposed to HomeKit as different kinds of sensors?
Thanks in advance

**Comment by alxscms** [comment]
Hi there, do you mean that you would want the plugin to be able to run as a child bridge in homebridge?

**Comment by LuOwl3** [comment]
Yes indeed… I feel like that could be quite helpful with the window and motion sensors so that they could also be used in automations yk?

**Comment by alxscms** [comment]
Oh ok that is something else. You actually want the window/door sensors + motion sensors exposed in HomeKit. It could be pretty cool actually but I don't know if that is possible actually. I guess I could get the battery level of each but that's pretty much it...

**Comment by LuOwl3** [comment]
Would you try?
